src/index.cts

```diff
@@ -1,20 +1,20 @@
 // This module is the CJS entry point for the library.
 
-// The Rust addon.
-import * as addon from './load.cjs'
+export { encrypt, encryptBulk, newClient, decryptBulk } from './load.cjs'
+import { decrypt as ffiDecrypt } from './load.cjs'
+
+declare const sym: unique symbol
+
+// Poor man's opaque type.
+export type Client = { readonly [sym]: unknown }
 
 // Use this declaration to assign types to the jseql's exports,
 // which otherwise by default are `any`.
 declare module './load.cjs' {
-  interface Client {}
-
   function newClient(encryptSchema?: string): Promise<Client>
   function encrypt(
     client: Client,
-    plaintext: string,
-    columnName: string,
-    tableName?: string,
-    context?: Context,
+    plaintext: EncryptPayload,
```

Comment:
`plaintext`, `columnName`, `tableName`, and `context` have been combined into a single arg of type `EncryptPayload`. `encrypt` takes a single `EncryptPayload` and `encryptBulk` takes an `EncryptPayload[]`.

Is this similar to what you had in mind @calvinbrewer? I can do something similar for `decrypt`/`decryptBulk` as well if that's helpful (but we'd only be combining `ciphertext` and `lockContext` into a `DecryptPayload`.